config/settings.yml

```diff
@@ -240,6 +240,9 @@ modules_appeals_api:
     appeals_channel_id: ''
   documentation:
     path_enabled_flag: false
+    wip_docs:
+      - supplemental_claims
```

Comment:
Only added supplemental_claims as an example for the rspec tag filtering. I know we've already put SC docs into prod :)